rollup/tracing/mux_tracer.go

```diff
@@ -0,0 +1,60 @@
+package tracing
+
+import (
+	"math/big"
+	"time"
+
+	"github.com/scroll-tech/go-ethereum/common"
+	"github.com/scroll-tech/go-ethereum/core/vm"
+	_ "github.com/scroll-tech/go-ethereum/eth/tracers/native"
+)
+
+type MuxTracer struct {
```

Comment:
Add comment

Reply:
resolved in https://github.com/scroll-tech/go-ethereum/pull/615/commits/acf23b72b5312edadf3ff4a9a1f6a3248ab47d75